distribution/kernel/pom.xml

```diff
@@ -186,6 +186,38 @@
                                         platform-osgi-condpermadmin-${project.version}.jar
                                     </destFileName>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.apache.servicemix.bundles</groupId>
+                                    <artifactId>org.apache.servicemix.bundles.xerces</artifactId>
+                                    <version>2.11.0_1</version>
+                                    <outputDirectory>
+                                        ${setup.folder}/endorsed
+                                    </outputDirectory>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.apache.servicemix.specs</groupId>
+                                    <artifactId>org.apache.servicemix.specs.saaj-api-1.3</artifactId>
+                                    <version>2.9.0</version>
+                                    <outputDirectory>${setup.folder}/endorsed</outputDirectory>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.apache.servicemix.specs</groupId>
+                                    <artifactId>org.apache.servicemix.specs.jaxb-api-2.2</artifactId>
+                                    <version>2.9.0</version>
+                                    <outputDirectory>${setup.folder}/endorsed</outputDirectory>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.apache.servicemix.specs</groupId>
+                                    <artifactId>org.apache.servicemix.specs.locator</artifactId>
+                                    <version>2.9.0</version>
+                                    <outputDirectory>${setup.folder}/lib</outputDirectory>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.apache.servicemix.specs</groupId>
+                                    <artifactId>org.apache.servicemix.specs.activator</artifactId>
+                                    <version>2.9.0</version>
```

Comment:
Can you pull all the 2.9.0 into a version variable?

Reply:
✏️ Is the servicemix.specs.version tied with the karaf version? Should there be a warning where the version is defined to not update it without upgrading Karaf?

Reply:
I don't think it's a tight coupling like some of the other transitive dependencies, but it _should_ be updated when Karaf is upgraded. Adding that now.